Drink and Vending Add-on/Patch 2: Electric Boogaloo (Youtubeboy139)

Youtubeboy139 opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

It adds drinks and vending stuff

## Why It's Good For The Game

It adds extra canned drinks, replaces the Nuka Cola bottle with something that makes sense, and adds a new cola vending machine

## Changelog
:cl:
-Adds canned drinks of existing as well as new reagents
-Adds Hades Hellfire mixed drink
-Adds Bepsi, Kvass, Lager, lagger_lager(WIP), IPA, Guinness
-Adds Bepsi Vending Machine
-Adds new Nuka Cola bottle sprite for vending machines and new glass sprite for mixed drink
-Adds Hairy Dairy mixed drink
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

Youtubeboy139 committed:
Add files via upload

Youtubeboy139 committed:
Add files via upload

Youtubeboy139 commented:
I couldn't figure out how to add files to an existing PR, wanted to integrate all the files incase I missed something and cause the previous 'not killing you error' so I deleted my fork, remade it, put all the files in it, and made a new PR... so here we are

TheSwain commented:
EIGHTY files changed??? i can't even review all of that. what was changed in ALL those files???

Youtubeboy139 commented:
> EIGHTY files changed??? i can't even review all of that. what was changed in ALL those files???

There are actually only a few changes. This project was already submitted a couple of months ago, but there was a supposed glitch where the Vending Machine that I added didn't tip over and kill you. I tried adding more files that I thought I forgot to the original Pull Request but the website refused to cooperate. So I scrapped the whole thing and did another one, this time including the entire folders where I changed files. In reality, there's maybe a dozen or so actual file changes, all of them commented with //ADDITION followed by the date that they were added. I do this so I can just Ctrl+F the code and find exactly what I added in case I needed to make changes. I apologize for the massive amount of files here, but I had to be sure that I added all the correct files needed to make this PR work.
